app/helpers/events_helper.rb

```diff
@@ -0,0 +1,14 @@
+module EventsHelper
+  def format_event_date(event)
+    return if event.startDate.blank?
```

Comment:
The camel case here makes me cringe - do you want me to update the API to spit out snake case? I think it's just a serialiser setting

Reply:
Agreed, it was done to maintain parity with the API. I'm in two minds whether the appropriate option is to amend the case here or in the API. It is orthoganol to these changes though so I'd suggest doing that in a separate PR?

Reply:
I would vote for changing the API - it's more than likely going to be consumed by Rails apps predominantly so I think it makes sense to change it there; I'll take a look into it